cmd/config/olm/olm.yml

```diff
@@ -0,0 +1,82 @@
+---
+global:
+  gc: {{.GC}}
+  gcMetrics: {{.GC_METRICS}}
+  measurements:
+    - name: podLatency
+    - name: containerResourceUsage
```

Comment:
What is this measurement? is it defined somewhere?

Reply:
Thanks! Not found in https://github.com/kube-burner/kube-burner/tree/main/pkg/measurements, removed it.